docs/software/apple/usage.mdx

```diff
@@ -3,6 +3,71 @@ id: usage
+
+Let us call the thing with the antenna a "node" for now.
+
+On all platforms the app uses the usual features of the OS, i.e. for notifications (in "Notifications|Meshtastic"), for timestamps (in "General|Language&Region|Date Format"), plus on the Mac there is extra functionality in the menu bar.
+
+The app itself has help in the form of tips for many situations, and we will not duplicate them here.
+
+## First App Start
+
+First, the node should be prepared https://meshtastic.org/docs/getting-started - otherwise the app will look pretty boring - as most of the functionality depends on a connected node.
+
+The app always opens the "Bluetooth" page first. If your node does not show up on the page, make it physically start up now - a simple way is to disconnect power (a.k.a. the USB cable) for a moment. The app should now feel the node and you read a 6-digit number from the node and key it into the app. Then the node should show up as "connected". A green symbol with the node's short name will now show up in the upper right corner - and this indicator will be on each page from now on.
+
+NB: You do not use the iPhone's (or iPad or Mac) settings to connect the node as a Bluetooth device, but you will find the node under Bluetooth when you are done here.
+
+## the App
+
+The bottom app bar has tabs "Messages", "Bluetooth", "Nodes", "Mesh Map" and "Settings".
+
+The __Messages__ tab has channels and direct messages (DMs), comparable to groups and users, where there is a default group, where people can be received you have not known before. Other channels can be configured and there is a channel that will not be visible here, the admin channel. 
+
+__Messages|DirectMessages__ has all nodes as seen in the "Nodes" page, but this time as communication partners, i.e. people and routers and maybe sensors. Makes sense since you might need to give commands to a router or request data from a sensor. The number of people for DMs will grow over time.
+
+The __Bluetooth__ tab has the connection status of your directly connected node. (over BLE) If your node does not show up for connecting, reboot it - it will respond to connect requests for a minute or so. If you want to physically connect to a different node, restart the app and connect here.
+
+__Nodes__ over time builds up a list of nodes your node has "seen". This is technical info about nodes. Do not be surprised by the time that is reported, like "seen in 71 years"; this comes from the node device, not from the Apple app. A proper message would be "seen 24 seconds ago", but some nodes send a wrong timestamp. What is really helpful here is the signal strength (in dbm) and the signal-to-noise ratio (SNR), both of which may be not very precise, but can give an estimation of how far you can send; or if there is an improvement when you use a different antenna (or a different antenna placement or orientation).
```

Comment:
"This is technical info about nodes."  Maybe this could be more specific.

I'd leave out:
"Do not be surprised by the time that is reported, like "seen in 71 years"; this comes from the node device, not from the Apple app. A proper message would be "seen 24 seconds ago", but some nodes send a wrong timestamp"
and
"both of which may be not very precise"